Bottom: 90123bc9172783ec76ea48574175bcc4512c6e36
Top:    8ef25628b90a637e6623e6144c2c434e7ab5cfcd
Author: Ian Kent <ikent@redhat.com>
Date:   2013-07-13 14:14:41 +0800

autofs-5.0.7 - add after sssd dependency to unit file

When using sss as a map source autofs can sometimes start before
sssd is ready.


---

diff --git a/CHANGELOG b/CHANGELOG
index 7749f01..2734fe3 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -55,6 +55,7 @@
 - fix add null check in parse_server_string().
 - don't override LDFLAGS in make rules.
 - fix a couple of compiler warnings.
+- add after sssd dependency to unit file.
 
 25/07/2012 autofs-5.0.7
 =======================
diff --git a/samples/autofs.service.in b/samples/autofs.service.in
index 056ab5e..777463d 100644
--- a/samples/autofs.service.in
+++ b/samples/autofs.service.in
@@ -1,6 +1,6 @@
 [Unit]
 Description=Automounts filesystems on demand
-After=network.target ypbind.service
+After=network.target ypbind.service sssd.service
 
 [Service]
 Type=forking
